bigquery_etl/cli/query.py

```diff
@@ -0,0 +1,144 @@
+"""bigquery-etl CLI query command."""
+
+import click
+import os
+from pathlib import Path
+import re
+import sys
+import string
+
+from bigquery_etl.metadata.parse_metadata import Metadata
+
+
+QUERY_NAME_RE = re.compile(
+    r"(?P<dataset>[a-zA-z0-9_]+)\.(?P<name>[a-zA-z0-9_]+)(?P<ver>_v[0-9]+)?"
+)
+
+
+def create_query_cli_group():
+    """Create the CLI group for the query command."""
+    group = click.Group(name="query", help="Creating/validating/deleting/... queries")
+    group.add_command(query_create_command)
+    return group
```

Comment:
Commands can consist of sub-commands, like in this case where I could imagine we want to have things like:
`bqetl query create ...`
`bqetl query delete ...`
`bqetl query schedule ...`
`bqetl query validate ...`